### PR TITLE
migrate formbuilder-product-page-prod to live-1

### PR DESCRIPTION
**Pull request by asmega**
- this is a copy paste of the existing `formbuilder-product-page-prod`
- i have not created the ECR repo yet

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/formbuilder-product-page-prod/00-namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: formbuilder-product-page-prod
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
```
Comment (digitalronin) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/formbuilder-product-page-prod/00-namespace.yaml]: Should "is-production" be "true" here?

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/formbuilder-product-page-prod/resources/ecr.tf [hunk 2]
```diff
@@ -0,0 +1,27 @@
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "eu-west-2"
+}
+
```
Comment (digitalronin) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/formbuilder-product-page-prod/resources/ecr.tf]: I'd recommend extracting lines 1-8 into a separate main.tf file

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/formbuilder-product-page-prod/resources/ecr.tf [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+module "formbuilder_product_page_ecr_credentials" {
+  source    = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=2.1"
```
Comment (digitalronin) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/formbuilder-product-page-prod/resources/ecr.tf]: In live-1 you can (and should) use the latest version of the module - currently "ref=3.1"

**Comment by asmega** [comment]
@digitalronin thanks for the feedback. I have made all the suggested changes. and yes that value should be true for the production flag

**Pull request merged**